doc/api/esm.md

```diff
@@ -1210,18 +1210,17 @@ a URL should be interpreted. This can be one of the following:
 ```js
 /**
  * @param {string} url
- * @param {object} context (currently empty)
- * @param {function} defaultGetFormat
- * @returns {object} response
- * @returns {string} response.format
+ * @param {Object} context (currently empty)
+ * @param {Function} defaultGetFormat
+ * @returns {Promise<{ format: string }>}
  */
 export async function getFormat(url, context, defaultGetFormat) {
-  if (someCondition) {
+  if (Math.random() > 0.5) { // Some condition.
     // For some or all URLs, do some custom logic for determining format.
     // Always return an object of the form {format: <string>}, where the
     // format is one of the strings in the table above.
     return {
-      format: 'module'
+      format: 'module',
```

Comment:
I'm pretty sure our linter doesn't like trailing commas.

Reply:
This just came up in a review not too long ago. The conclusion was that yes, they are desirable.

https://github.com/nodejs/node/blob/5007611294bfcfe7af8648a69bd19d1b014c84bd/.eslintrc.js#L83

Btw, these snippets are also being linted by ESLint too.